lib/pry/history.rb

```diff
@@ -47,14 +47,16 @@ def push(line)
       unless line.empty? || (@history.last && line == @history.last)
         @pusher.call(line)
         @history << line
-        @saver.call(line) if Pry.config.history.should_save
+        if !should_ignore?(line) && Pry.config.history.should_save
+          @saver.call(line)
+        end
       end
       line
     end
     alias << push
 
     # Clear this session's history. This won't affect the contents of the
-    # history file.
+    #   history file.
```

Comment:
I actually meant to say that it would be nice to indent the comment only on lines 85 and 95, where you use the `@return` YARD tag. It's just a convention for YARD. Other comments like this one and on line 93 don't need this :)